test/integration/setup/TokenRules.js

```diff
@@ -0,0 +1,54 @@
+const chai = require('chai'),
+  Web3 = require('web3');
+
+const TokenRules = require('../../../lib/setup/TokenRules'),
+  config = require('../../utils/configReader'),
+  Web3WalletHelper = require('../../utils/Web3WalletHelper'),
+  AbiBinProvider = require('../../../lib/AbiBinProvider');
+
+const auxiliaryWeb3 = new Web3(config.gethRpcEndPoint),
+  web3WalletHelper = new Web3WalletHelper(auxiliaryWeb3),
+  abiBinProvider = new AbiBinProvider(),
+  assert = chai.assert;
+
+describe('TokenRules', async function() {
+
+  let options = {
+    from: config.deployerAddress,
+    gasPrice: config.gasPrice,
+    gas: '8000000'
```

Comment:
let's use the gas value from configReader. Avoid hardcoding values in test files